discuss/customizations/mail_alias_template.rst

```diff
@@ -0,0 +1,91 @@
+======================================
+Email Parsing via Mail Alias Templates
+======================================
+
+Configure mail alias template to be able to parse email content that is used to create and update related records.
+
+Configuration
+=============
+
+To be able to use mail alias templates, :code:`mail_alias_template` module must be installed first.
+
+Then can open or create wanted mail alias (in :menuselection:`Settings --> Technical --> Email --> Aliases`). To be able to use template, check :code:`Use Template` option.
+
+First need to specify :code:`Line Pattern`. Here special placeholders :code:`_KEY_` and :code:`_VAL_` must be specified on this field. These placeholders are used to know where to expect key and value when parsing email message. Line Pattern Also consists of optional Prefix (that goes before key/value pair) and suffix (that goes after key/value pair) and mandatory separator (that indicates how to distinguish key from value).
+
+If Prefix and Suffix would be used, pattern example could be: :code:`[_KEY_] {_VAL_}`, where :code:`[` would be prefix, :code:`] {` would be separator and :code:`}` would be suffix.
+
+Second thing that is needed - to add actual mapping lines. On :code:`Key` field specify key that is expected to be found on email message and on :code:`Field` field specify to which field map parsed value from message.
+
+.. image:: media/mail_alias_template_cfg.png
+    :align: center
+
+Map Line Options
+================
+
+.. image:: media/mail_alias_template_map_line.png
+    :align: center
+
+Update record with another email message
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Related record update can be done in two ways:
+
+* New message is reply to message that was sent from Odoo or message has special Message-ID header that lets Odoo identify related record.
```

Comment:
"Replying to existing message that was sent from Odoo or message that has a special Message-ID header that odoo uses in order to identify related record"